<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgWidget</class>
 <widget class="QWidget" name="ImgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>581</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>535</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddToDB">
       <property name="text">
        <string>AddToDB</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/addToDB.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddFromTable">
       <property name="text">
        <string>Просмотр</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/openViewer.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddList">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/plusMany.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddFromTable">
   <property name="text">
    <string>Просмотр</string>
   </property>
   <property name="toolTip">
    <string>Просмотр из таблицы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
